1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,24 @@
-// Write your code here
+
+var  i =0; //start point 
+let images = []; //images array 
+let time = 3000; // time between switch
+
+//Image List 
+images[0] = 'https://images.unsplash.com/photo-1644782955074-13b15e5b5047?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80';
```

Comment:
as opposed to adding these to the images array, you can just do 

```
images = ['https://images.unsplash.com/photo-1644782955074-13b15e5b5047?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80',
'https://images.unsplash.com/photo-1587463272361-565200f82b33?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=387&q=80',
```
and so on :)